xml/admin_monitoring_alerting.xml

```diff
@@ -189,717 +133,8 @@
   <para>
    The &alertmanager; handles alerts sent by the &prometheus; server. It takes
    care of deduplicating, grouping, and routing them to the correct receiver.
-   It also takes care of silencing of alerts. &alertmanager; is configured via
-   the command line flags and a configuration file that defines inhibition
-   rules, notification routing and notification receivers.
+   It also takes care of silencing of alerts.
```

Comment:
Alerts can be silenced using the alertmanager, though there's functionality built into the (ceph) dashboard that enables the user to do that from the (ceph) dashboard.